### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@owncloud-bot haha - funny you

**Comment by DeepDiver1975** [comment]
@MorrisJobke @LukasReschke @PVince81 @icewind1991 please review - THX

File: lib/private/app/appmanager.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ private function getInstalledAppsValues() {
 	/**
 	 * List all installed apps
 	 *
-	 * @return string[]
+	 * @return integer[]
```
Comment (MorrisJobke) [inline lib/private/app/appmanager.php]: Really?

File: lib/private/db/connection.php [hunk 2]
```diff
@@ -214,7 +214,7 @@ public function executeUpdate($query, array $params = array(), array $types = ar
 	 * columns or sequences.
 	 *
 	 * @param string $seqName Name of the sequence object from which the ID should be returned.
-	 * @return string A string representation of the last inserted ID.
+	 * @return integer A string representation of the last inserted ID.
```
Comment (MorrisJobke) [inline lib/private/db/connection.php]: String is correct - that is what DBAL returns  - The actual wrong documentation is here: https://github.com/owncloud/core/blob/682821c71ed2af297170294632931be1ac769f25/lib/private/db/adapter.php#L46-L46

File: lib/private/helper.php [hunk 3]
```diff
@@ -591,7 +594,7 @@ public static function findBinaryPath($program) {
 	 *
 	 * @param string $path
 	 * @param \OCP\Files\FileInfo $rootInfo (optional)
-	 * @return array
+	 * @return string
```
Comment (MorrisJobke) [inline lib/private/helper.php]: Wrong.

File: lib/private/user/session.php [hunk 4]
```diff
@@ -143,7 +143,7 @@ public function setUser($user) {
 	/**
 	 * get the current active user
 	 *
-	 * @return \OCP\IUser|null Current user, otherwise null
+	 * @return null|User Current user, otherwise null
```
Comment (MorrisJobke) [inline lib/private/user/session.php]: Even if correct I would keep the interface here

File: lib/private/util.php [hunk 5]
```diff
@@ -1362,7 +1362,7 @@ public static function clearOpcodeCache() {
 	 * Normalize a unicode string
 	 *
 	 * @param string $value a not normalized string
-	 * @return bool|string
+	 * @return string
```
Comment (MorrisJobke) [inline lib/private/util.php]: `Normalizer::normalize()` could also return `boolean` https://github.com/owncloud/3rdparty/blob/678d52a49ed0a8458a29a51d4343ad6a7d83c985/patchwork/utf8/src/Patchwork/PHP/Shim/Normalizer.php#L54-L54

**Comment by MorrisJobke** [comment]
:-1: This is not a good auto-fix. I guess we need to report some bugs to scrutinizer.

**Comment by MorrisJobke** [comment]
cc @DeepDiver1975

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Kill it?!

**Pull request closed without merging**